### ISSUE TITLE
Questions on Evaluation on Human3.6M

### ISSUE THREAD

**Issue by avegetablechicken** [ZhengyiLuo/PHC#154]
Hi Zhengyi, thank you for the excellent work!
I am wondering how you actually evaluate the performance on downstream tasks like tracking mocap data or noisy inputs. When you evaluate on a dataset will you finetune the policy on it?

**Comment by ZhengyiLuo** [comment]
Hi! No all policies are trained on the training split of AMASS dataset only. 

H3.6M and AMASS test set are only used for evaluation.

**Comment by avegetablechicken** [comment]
Thanks for your reply. Then which configuration & weights should I use to reproduce experiments on Human3.6M test split? @ZhengyiLuo

**Comment by avegetablechicken** [comment]
I use phc_comp_3 checkpoints and reproduced 99.9% tracking succ rate on AMASS train split, but only 17.4% on filtered HybrIK+MeTRAbs estimated motion. I have forced the agent on the ground and can reach  90% succ rate after training for 3000 epoch.

**Comment by ZhengyiLuo** [comment]
> Thanks for your reply. Then which configuration & weights should I use to reproduce experiments on Human3.6M test split? [@ZhengyiLuo](https://github.com/ZhengyiLuo)

For reproducing the H3.6M data, please refer to the ICCV weights. Is there any gap you are seeing? 

> 17.4% on filtered HybrIK+MeTRAbs estimated motion

Could you elaborate on what the dataset is for this setting?  `I have forced the agent on the ground and can reach 90% succ rate after training for 3000 epoch.` Is this referring to training?

**Comment by avegetablechicken** [comment]
I followed the preprocessing procedure referred in the paper to get estimated motion on Human3.6M. Both policies of "phc_shape_mcp_iccv" and "phc_comp_3" gives  17+% tracking success rates on test split of Human3.6M. I trained "phc_comp_3" policy on Human3.6M test split for 3000 epochs and the success rate becomes 94%.

**Comment by avegetablechicken** [comment]
phc_shape_mcp_iccv on Human3.6M test split:
<img width="1786" height="101" alt="Image" src="https://github.com/user-attachments/assets/133a1b95-8536-46eb-af00-78139d3e6f98" />
phc_kp_mcp_iccv on Human3.6M test split:
<img width="1013" height="60" alt="Image" src="https://github.com/user-attachments/assets/a487251e-0621-428f-83da-029ae7c6114a" />
phc_comp_3 on on Human3.6M test split: 
<img width="1757" height="92" alt="Image" src="https://github.com/user-attachments/assets/bac80e31-5166-4b4e-924a-1f92e59a0862" />

phc_comp_3 on AMASS train split:
<img width="1757" height="96" alt="Image" src="https://github.com/user-attachments/assets/ad7deb20-e838-4cc1-b390-c3621c70c084" />

**Comment by avegetablechicken** [comment]
Sorry I made a mistake on models for evaluation. Fine-tuning on Human3.6M test split cannot get a 90+% success rate. It seems that training "phc_comp_3" can only get a 44% result (48% for "phc_kp_mcp_iccv").
Actually I trained "pnn_big" from scratch by myself and this model can get a 90+% after fine-tuning. Below is its training log on AMASS train split & Human3.6m test split:

<img width="409" height="237" alt="Image" src="https://github.com/user-attachments/assets/9f543412-abff-41ee-95d7-52e507f2c6ee" />

<img width="402" height="238" alt="Image" src="https://github.com/user-attachments/assets/ababfc1a-7824-42ef-9f74-aa3e943f03c9" />

This model's success rate without fine-tuning on Human3.6M test split is very low as well.
